src/language/mongoDBService.ts

```diff
@@ -153,47 +170,39 @@ export default class MongoDBService {
             this._extensionPath,
             'dist',
             languageServerWorkerFileName
-          ),
-          {
-            // The workerData parameter sends data to the created worker.
-            workerData: {
-              codeToEvaluate: executionParameters.codeToEvaluate,
-              connectionString: this._connectionString,
-              connectionOptions: this._connectionOptions,
-            },
-          }
+          )
         );
-
         this._connection.console.log(
-          `MONGOSH execute all body: "${executionParameters.codeToEvaluate}"`
+          `WORKER thread is created on path: ${this._extensionPath}`
         );
 
-        // Evaluate runtime in the worker thread.
-        worker.postMessage(ServerCommands.EXECUTE_ALL_FROM_PLAYGROUND);
-
-        // Listen for results from the worker thread.
-        worker.on(
-          'message',
-          (response: [Error, ShellExecuteAllResult | undefined]) => {
-            const [error, result] = response;
-
-            if (error) {
-              const printableError = formatError(error);
-
-              this._connection.console.error(
-                `MONGOSH execute all error: ${util.inspect(printableError)}`
-              );
-              void this._connection.sendNotification(
-                ServerCommands.SHOW_ERROR_MESSAGE,
-                printableError.message
-              );
-            }
-
-            void worker.terminate().then(() => {
-              resolve(result);
-            });
-          }
-        );
+        worker?.on('error', (error) => {
```

Comment:
When can `worker` be false-y here?

Reply:
Refactored as we discussed during 1-on-1 to always send back a message and not throw the error.